### DISCUSSION TITLE
Opacity transition did not seem to be applied correctly the first time.

### DISCUSSION THREAD

**Discussion by YangSpring114** [AvaloniaUI/Avalonia#14269] [Q&A]
https://github.com/AvaloniaUI/Avalonia/assets/99802662/cdf1755a-aebb-4022-a10a-aa04396b476b

![{E9BBE58D-1A86-46aa-A161-B68EE033E94F}](https://github.com/AvaloniaUI/Avalonia/assets/99802662/c5fd79d6-b1bf-4e74-8ffd-c16aa906defa)

![{980D5FDD-308F-4906-A191-31C0B863BBE5}](https://github.com/AvaloniaUI/Avalonia/assets/99802662/864fa6e4-1427-47f5-825c-5dd6697f7067)
As for the title, the transition was not applied correctly, and I am not sure if it is my problem or Avalonia's bug. The attached images and videos contain the code and actual operation situation.

Avalonia version:11.0.6
OS:Windows 11

**Comment by kyuranger** [comment]
Have you used a break point to check the code
`Dispatcher.UIThread.Post(() => _contentPanel.Opacity = 1);`
ran or not?
